.evergreen/run-static-checks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+export JAVA_HOME=/opt/java/jdk11
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Compiling and running checks"
+
+# We always compile with the latest version of java
```

Comment:
What does this mean, given the JAVA_HOME above?

Reply:
Copy pasta - removed